make.ps1

```diff
@@ -6,51 +6,45 @@
 function All-Command 
```

Comment:
```suggestion
function All-Command 
{
	Dependencies-Command 

	if (!(Test-Path "C:\Program Files\dotnet\dotnet.exe"))
	{
		DotnetNotFound
		return
	}

	dotnet.exe build /nr:false
	if ($lastexitcode -ne 0)
	{
		echo "Build failed. If just the development tools failed to build, try installing Visual Studio. You may also still be able to run the game."
	}
	else
	{
		echo "Build succeeded."
	}
}
```

Reply:
This one actually seems to have worked. gj!